### ISSUE TITLE
Using DrawerSlideTransition should resize the side view controllers to the available width

### ISSUE THREAD

**Issue by dvkch** [kawoou/KWDrawerController#18]
First off, really good work, it's a really clear project and easy to setup!

I have a suggestion: when using a `DrawerSlideTransition`, since no background is visible (like `DrawerFloatTransition` for instance), it would be great if the child view controllers were resized to fit the available width.

I have been able to make this by updating a bit `DrawerContent.swift` :

```
    internal func updateView() {
        let width = CGFloat(drawerWidth)
        contentView.frame.size.width = width
        drawerOffset = 0.0
        
        guard let superView = contentView.superview else { return }
        if drawerSide == .right {
            drawerOffset = superView.frame.width - width
        }
        viewController.view.frame = CGRect(
            x: 0,
            y: 0,
            width: width,
            height: superView.frame.height
        )
    }
```

This may not be perfect to obtain the needed behaviour, and this affects all other transitions. Would you know a way to achieve this behaviour with a custom transition or a an additional setting?

I also noticed it's not possible de create custom transitions because the methods of `DrawerTransition` are not marked `open`.

**Comment by kawoou** [comment]
Thank you for your good suggestion.

First, the proposed `DrawerContent.swift` file has been modified in #24.

Next, i will modify the part that is not `open`.
Thank you for letting know what I missed.

**Comment by kawoou** [comment]
@dvkch Fixed access levels issue #27.
